x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_schema.tsx

```diff
@@ -300,28 +361,113 @@ export const EngineSchema: React.FC = () => {
       }}
       engineName={engineName}
     >
-      <EuiFlexGroup direction="column" gutterSize="l">
-        <EuiFlexGroup>
-          <EuiSwitch
-            label={i18n.translate(
-              'xpack.enterpriseSearch.content.engine.schema.onlyShowConflicts',
-              {
-                defaultMessage: 'Only show conflicts',
-              }
-            )}
-            checked={onlyShowConflicts}
-            onChange={toggleOnlyShowConflicts}
+      <>
+        <EuiFlexGroup direction="column" gutterSize="l">
+          <EuiFlexGroup>
+            <EuiSwitch
+              label={i18n.translate(
+                'xpack.enterpriseSearch.content.engine.schema.onlyShowConflicts',
+                {
+                  defaultMessage: 'Only show conflicts',
+                }
+              )}
+              checked={onlyShowConflicts}
+              onChange={toggleOnlyShowConflicts}
+            />
+            <EuiFlexGroup justifyContent="flexEnd" alignItems="center">
+              <EuiFlexItem grow={false}>
+                {i18n.translate('xpack.enterpriseSearch.content.engine.schema.filters.label', {
+                  defaultMessage: 'Filter By',
+                })}
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiPopover
+                  button={filterButton}
+                  isOpen={isFilterByPopoverOpen}
+                  closePopover={() => setIsFilterByPopoverOpen(false)}
+                  panelPaddingSize="m"
```

Comment:
I'd me curious how this looks with `s` instead. the popover seems a big big to me and this may be why. @julianrosado ?

Reply:
Yup! Technically this is flush per https://eui.elastic.co/#/forms/filter-group#multi-select but I know Saarika did that to accommodate the clear all filters button. Am connecting with EUI folks to check if there's an existing pattern for clearing all filters without it being a combo box before I suggest alternatives. INVESTIGATING!